src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/burdenEstimateRepository/BurdenEstimateWriterTests.kt

```diff
@@ -211,4 +213,53 @@ class BurdenEstimateWriterTests : BurdenEstimateRepositoryTests()
         }
     }
 
+    @Test
+    fun `duplicate rows throw error`()
```

Comment:
This functionality was being implicitly tested via the repository tests but naturally belonged here anyway.